frontend/app/model/PackagePromo.scala

```diff
@@ -0,0 +1,8 @@
+package model
+import com.netaporter.uri.Uri
+
+object PackagePromo {
+
+  // the call-to-action button on the package promo view element
+  case class CtaButton(text: String, to: Uri, attributes: Map[String, String])
```

Comment:
maybe this case class doesn't need to be wrapped in an object? not sure what extra benefit we get that isn't already provided by Scala's package system